### PR TITLE
sentry-cocoa: adds SPM. adds heading to install methods.

### PR DESCRIPTION
**Pull request by fetzig**
SPM support will come soon.

Updated docs accordingly. Added heading to every install method, since we have 3 distinct install methods now.

### CODE REVIEW

File: src/collections/_documentation/clients/cocoa/index.md [hunk 1]
```diff
@@ -43,6 +47,14 @@ Run `carthage update` to download the framework and drag the built _Sentry.frame
 
 We also provide a pre-built version for every release which can be downloaded at [releases on GitHub](https://github.com/getsentry/sentry-cocoa/releases).
 
+### SPM | Swift Package Manage
+
+Starting with sentry-cocoa version `4.4.3`, we support Swift Package Manager.
+
+To integrate Sentry into your xCode project using SPM, open your App in xCode and open File -> Swift Packages -> Add Package Dependency. Then add sentry-cocoa by entering the git repo url `https://github.com/getsentry/sentry-cocoa.git`, and select a version (or brach) on the next page.
```
Comment (ydnar) [inline src/collections/_documentation/clients/cocoa/index.md]: Typo: brach —> branch

File: src/collections/_documentation/clients/cocoa/index.md [hunk 2]
```diff
@@ -15,10 +15,12 @@ Getting started with Sentry is a three step process:
 <!-- WIZARD -->
 ## Installation {#install}
 
-The SDK can be installed using [CocoaPods](http://cocoapods.org) or [Carthage](https://github.com/Carthage/Carthage). This is the recommended client for both Swift and Objective-C.
+The SDK can be installed using [CocoaPods](http://cocoapods.org), [Carthage](https://github.com/Carthage/Carthage) or [Swift Package Manager](https://swift.org/package-manager/). This is the recommended client for both Swift and Objective-C.
```
Comment (ydnar) [inline src/collections/_documentation/clients/cocoa/index.md]: Might want to throw an Oxford comma in there before “or Swift Package Manager.”

File: src/collections/_documentation/clients/cocoa/index.md [hunk 3]
```diff
@@ -43,6 +47,14 @@ Run `carthage update` to download the framework and drag the built _Sentry.frame
 
 We also provide a pre-built version for every release which can be downloaded at [releases on GitHub](https://github.com/getsentry/sentry-cocoa/releases).
 
+### SPM | Swift Package Manage
```
Comment (ydnar) [inline src/collections/_documentation/clients/cocoa/index.md]: Swift Package Manager

(So the anchor is clean)

File: src/collections/_documentation/clients/cocoa/index.md [hunk 4]
```diff
@@ -43,6 +47,14 @@ Run `carthage update` to download the framework and drag the built _Sentry.frame
...
+
+Starting with sentry-cocoa version `4.4.3`, we support Swift Package Manager.
+
+To integrate Sentry into your xCode project using SPM, open your App in xCode and open File -> Swift Packages -> Add Package Dependency. Then add sentry-cocoa by entering the git repo url `https://github.com/getsentry/sentry-cocoa.git`, and select a version (or brach) on the next page.
+
+NOTE: Version tags or branches need to have the Package.swift file in it or xCode won't be able to install the package. Thus versions previous to `4.4.3` can't be installed via SPM!
```
Comment (ydnar) [inline src/collections/_documentation/clients/cocoa/index.md]: Does this doc consistently use “xCode” versus “Xcode?”

File: src/collections/_documentation/clients/cocoa/index.md (hunk 1 above)
Comment (fetzig) [reply src/collections/_documentation/clients/cocoa/index.md]: done

File: src/collections/_documentation/clients/cocoa/index.md (hunk 2 above)
Comment (fetzig) [reply src/collections/_documentation/clients/cocoa/index.md]: I do :)

File: src/collections/_documentation/clients/cocoa/index.md (hunk 3 above)
Comment (fetzig) [reply src/collections/_documentation/clients/cocoa/index.md]: true

File: src/collections/_documentation/clients/cocoa/index.md (hunk 4 above)
Comment (fetzig) [reply src/collections/_documentation/clients/cocoa/index.md]: thanks

File: src/collections/_documentation/clients/cocoa/index.md [hunk 5]
```diff
@@ -43,6 +47,14 @@ Run `carthage update` to download the framework and drag the built _Sentry.frame
 
 We also provide a pre-built version for every release which can be downloaded at [releases on GitHub](https://github.com/getsentry/sentry-cocoa/releases).
 
+### Swift Package Manage
```
Comment (ydnar) [inline src/collections/_documentation/clients/cocoa/index.md]: Forgot “r”

**Comment by fetzig** [comment]
@cameronmcefee sorry, but I must have deleted the "Manager" within "Swift Package Manager". your typo fix made me see that. fixed that.

**Comment by cameronmcefee** [comment]
@fetzig 👍 Looks good to me

**Pull request merged**